web/frontend/src/pages/election/Show.tsx

```diff
@@ -141,14 +169,20 @@ const ElectionShow: FC = () => {
 
```

Comment:
I'm not sure but this might be missing, since it's not set to `None` anywhere even if it successfully initializes ?
```suggestion
          setOngoingAction(OngoingAction.None);
```

Reply:
correct !